probe/host/reporter.go

```diff
@@ -23,7 +23,6 @@ const (
 	Load1         = "load1"
 	CPUUsage      = "host_cpu_usage_percent"
 	MemoryUsage   = "host_mem_usage_bytes"
-	ScopeVersion  = "host_scope_version"
```

Comment:
I'd alias that as `ScopeVersion = report.HostScopeVersion`, which requires fewer code changes and keeps the call sites shorter.